Formula/esbuild.rb

```diff
@@ -0,0 +1,31 @@
+require "language/node"
+
+class Esbuild < Formula
+  desc "Extremely fast JavaScript bundler and minifier"
+  homepage "https://esbuild.github.io/"
+  url "https://registry.npmjs.org/esbuild/-/esbuild-0.12.21.tgz"
+  sha256 "d0c130384288f48361c826a6ac18af61eacd3db90ce0a89eaa575cf753b37e78"
+  license "MIT"
+
+  depends_on "node"
+
+  def install
+    system "npm", "install", *Language::Node.std_npm_install_args(libexec)
+    bin.install_symlink Dir["#{libexec}/bin/*"]
+  end
+
+  test do
+    (testpath/"app.jsx").write <<~EOS
+      import * as React from 'react'
+      import * as Server from 'react-dom/server'
+
+      let Greet = () => <h1>Hello, world!</h1>
+      console.log(Server.renderToString(<Greet />))
+    EOS
+
+    system "npm", "install", "react", "react-dom"
```

Comment:
These are just two test dependencies, any idea about how to bypass the audit check?